tools/scripts/upgrades/upgrade_tx_v0.1.25_alpha.json

```diff
@@ -0,0 +1,15 @@
+{
+  "body": {
+    "messages": [
+      {
+        "@type": "/cosmos.upgrade.v1beta1.MsgSoftwareUpgrade",
+        "authority": "pokt10d07y265gmmuvt4z0w9aw880jnsr700j8yv32t",
+        "plan": {
+          "name": "v0.1.25",
+          "height": "UPDATE_ME",
```

Comment:
Missing

Reply:
It was in the upgrade list doc; fixed in https://github.com/pokt-network/poktroll/commit/c34256e6a33cf80e575a8a684260c10bb759b7b4